packages/strapi-design-system/src/Link/Link.stories.mdx

```diff
@@ -1,114 +1,44 @@
 <!--- Link.stories.mdx --->
 
-import { Meta, Story, Canvas } from '@storybook/addon-docs';
+import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
 import { ArgsTable } from '@storybook/addon-docs';
 import ChevronRight from '@strapi/icons/ChevronRight';
 import ArrowLeft from '@strapi/icons/ArrowLeft';
 import { Link } from './Link';
 import { Stack } from '../Stack';
-import Strapi from '@strapi/icons/Strapi';
 
 <Meta title="Design System/Components/Link" component={Link} />
 
-# Link
-
-Links are used to send users to another page or an anchor. They are used for external navigation.
-
-**Best practices**
-
-- Use TextButton component if an action needs to be triggered within the admin panel (ex: Opening a modal ).
-- Clicking on a Link most of the time opens a new tab.
-- Links should always be used with an icon. If it is an external link, use the ExternalLink icon accordingly.
-- Links text should be concise and describe their purpose. Users need to know exactly what they are about to do.
-- Use capital letters.
-
-[View source](https://github.com/strapi/design-system/tree/main/packages/strapi-design-system/src/Link)
-
-## Imports
-
-```js
-import { Link } from '@strapi/design-system/Link';
-```
-
-## Usage
-
-Links are used for external navigation. Moreover, they should be associated with an icon to indicate its external navigation purpose.
-
-### Base
-
-The simplest representation of the Link component.
+    @@ -39,32 +40,75 @@ The simplest representation of the Link component.
 
 <Canvas>
   <Story name="base">
     <Stack spacing={5}>
-      <Link isExternal href="https://strapi.io/">
-        External link
-      </Link>
-    </Stack>
-  </Story>
-</Canvas>
-
-### Disabled link
-
-Depending on the permissions a user have or the status of an action, a link can be unreachable or unavailable yet.
-
-<Canvas>
-  <Story name="disabled">
-    <Stack spacing={5}>
-      <Link isExternal href="https://strapi.io/" disabled>
-        Disabled link
-      </Link>
-    </Stack>
-  </Story>
-</Canvas>
-
-### Start & end icons
-
-Display icons on the left or right side.
-
-<Canvas>
-  <Story name="icons">
-    <Stack spacing={5}>
       <div>
-        <Link isExternal href="https://strapi.io/" startIcon={<Strapi />}>
-          Strapi
+        <Link to="https://strapi.io/">External link</Link>
```

Comment:
could we replace the to props with an href prop for the external links? those are breaking
<img width="1189" alt="image" src="https://user-images.githubusercontent.com/71838159/158582810-db6d0c11-bcda-44ac-b131-bfb68286cc48.png">